src/applications/disability-benefits/all-claims/pages/form0781/officialReport.js

```diff
@@ -7,42 +8,128 @@ import {
   officialReportPageTitle,
   officialReportsDescription,
   reportTypesQuestion,
+  reportTypesHint,
   otherReportTypesQuestion,
   otherReportTypesExamples,
+  validateReportSelections,
+  // reportTypeValidationError,
+  // showConflictingAlert,
 } from '../../content/officialReport';
 import {
   titleWithTag,
   form0781HeadingTag,
   mentalHealthSupportAlert,
 } from '../../content/form0781';
 import { arrayBuilderEventPageTitleUI } from '../../utils/form0781';
-import { OFFICIAL_REPORT_TYPES } from '../../constants';
+import {
+  OFFICIAL_REPORT_TYPES_SUBTITLES,
+  MILITARY_REPORT_TYPES,
+  OTHER_REPORT_TYPES,
+} from '../../constants';
 
-export default {
+const pageTitleWithTag = titleWithTag(
+  officialReportPageTitle,
+  form0781HeadingTag,
+);
+
+export const officialReport = {
   uiSchema: {
     ...arrayBuilderEventPageTitleUI({
-      title: titleWithTag(officialReportPageTitle, form0781HeadingTag),
+      title: pageTitleWithTag,
       editTitle: 'official report details',
     }),
-    'ui:description': officialReportsDescription,
-    reports: checkboxGroupUI({
+    'ui:description': officialReportsDescription(),
+    // 'view:conflictingResponseAlert': {
+    //   'ui:description': reportTypeValidationError,
+    //   'ui:options': {
+    //     hideIf: (formData, index, fullData) => {
+    //       const data = fullData.events?.[index];
+    //       return showConflictingAlert(data) === false;
+    //     },
+    //   },
+    // },
```

Comment:
Alerts on this page are disabled until `hideIf` functionality within the Array Builder Pattern has been corrected.